elide-core/src/main/java/com/yahoo/elide/graphql/ModelBuilder.java

```diff
@@ -230,9 +212,11 @@ public String parseLiteral(Object o) {
                 .dataFetcher(dataFetcher)
                 .type(customIdType));
 
+        // Additional special fields
         builder.field(newFieldDefinition()
-                    .name("__meta")
-                    .type(metaObject));
+                .name(getTotalRecordKey(entityName))
```

Comment:
We should have one of these for each relationship.

Reply:
Actually - we may need to support something more like Relay's style of pagination